cobalt/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -769,6 +769,12 @@ public boolean start() {
 
   @SuppressWarnings("unused")
   @UsedByNative
+  private boolean restart() {
+    // Restart MediaCodec after flush().
+    return start(null);
+  }
+
+  @SuppressWarnings("unused")
   public boolean start(CreateMediaCodecBridgeResult outCreateMediaCodecBridgeResult) {
```

Comment:
nit: Consider marking `outCreateMediaCodecBridgeResult` Nullable. Not sure if we currently use a nullness checker but we may in the future.

go/java-practices/null#annotations